bundles/org.openhab.binding.flicbutton/README.md

```diff
@@ -0,0 +1,119 @@
+
+```
+Bridge flicbutton:flicd-bridge:mybridge [ hostname="<YOUR_HOSTNAME>",  port=<YOUR_PORT>] {
+    Thing button myflic1 "<YOUR_LABEL>" [address ="<MAC_ADDRESS>"]
+    Thing button myflic2 "<YOUR_LABEL>" [address ="<MAC_ADDRESS>"]
+    ...
+}
+```
+
+You can lookup the MAC addresses of your buttons within the inbox of the UI. You're free to choose any label you like for your button.
+
+## Channels
+
+| Channel ID                | Channel Type             | Item Type                | Description                    |
+| ------------------------- | ------------------------ | --------------------------| ------------------------------ |
+| rawbutton                | [System Trigger Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-trigger-channel-types) `system.rawbutton`  | Depends on the  [Trigger Profile](https://www.openhab.org/docs/configuration/items.html#profiles) used | Raw Button channel that triggers `PRESSED` and `RELEASED` events, allows to use openHAB profiles or own implementations via rules to detect e.g. double clicks, long presses etc.  |
+| button                   | [System Trigger Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-trigger-channel-types) `system.button`    | Depends on the [Trigger Profile](https://www.openhab.org/docs/configuration/items.html#profiles) used | Button channel that is using Flic's implementation for detecting long, short or double clicks. Triggers `SHORT_PRESSED`,`DOUBLE_PRESSED` and `LONG_PRESSED` events.   |
+| battery-level            | [System State Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) `system.battery-level`     | Number | Represents the battery level as a percentage (0-100%).
+
+
+## Example
+
+### Initial setup
+
+1. Setup and run flicd as described in [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci). Please consider that you need a seperate Bluetooth adapter. Shared usage with other Bluetooth services (e.g. Bluez) is not possible.
+1. Connect your buttons to flicd using the simpleclient as described in [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci). Flicd has to run in background the whole time, simpleclient can be killed after you successfully tested the button connection.
+1. Add a flicd-bridge via UI or Textual Configuration. Please consider that flicd does only accept connections from localhost by default, to enable remote connections from openHAB you have to use the `--server-addr` parameter as described in [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci).
+1. When the bridge is online, buttons newly added via simpleclient will automatically get discovered via background Discovery. To discover buttons that were set up before the Binding setup, please run an active scan.
```

Comment:
New lines needed if possible. Suggest using dot points as the numbers are broken and each point has a number 1. showing.

Reply:
Where do you see they're not working? Nest uses same syntax without problems: https://www.openhab.org/addons/bindings/nest/

Reply:
This is my lack of knowledge on markdown, ignore my comment as the Nest works fine and is the same way as you wrote.